setup.py

```diff
@@ -20,14 +20,18 @@ def run(self):
         super().run()
         if not self.dry_run:
             try:
-                _run(["tar", "xjf", "htslib.tar.bz2", "-C", "build"])
-                _run(["tar", "xjf", "bcftools.tar.bz2", "-C", "build"])
-                _run(["tar", "xjf", "samtools.tar.bz2", "-C", "build"])
-                _run(["./configure"], cwd="build/htslib")
-                _run(["./configure"], cwd="build/samtools")
-                _run(["make"], cwd="build/htslib")
-                _run(["make"], cwd="build/bcftools")
-                _run(["make"], cwd="build/samtools")
+                for tool in ['samtools', 'htslib', 'bcftools']:
+                    if not os.path.exists(f'build/{tool}'):
```

Comment:
I realize I've been flip-flopping here, but should we just download always? Like if we upgrade the version again (just by incrementing the number) the build won't change because the old version of the tool already exists.

Up to you here and I think practically it doesn't really matter, but wanted to mention it at least.

Reply:
Good point.  I think I'll keep it this way for now, until I think of a better way.  Thanks for the review!